dashboard/tests/run_ui_tests.sh

```diff
@@ -12,6 +12,7 @@ trap clean_up EXIT
 
 echo "Installing cypress"
 if [ -n "$BUILDKITE" ]; then
+  apt update -y --fix-missing
```

Comment:
does the ci base image runs `apt update`? you can probably just make a small change (like add a comment line) to the related ci base image's dockerfile, invalidate the cache, and then get it fix?

at least maybe add this line `apt update` in the dockerfile, so the latest will be cached (and we do need to keep `apt update -y` on all future test jobs)?